### ISSUE TITLE
Unable to type fractional numbers for Workflow parameters in the New Task menu

### ISSUE THREAD

**Issue by ktaletsk** [usnistgov/WIPP-frontend#143]
**Summary**

When creating new Task in the Workflow, if one of the parameters is "number", then it is impossible to type fractional number in the corresponding UI field, as it only accepts integers. However, it is possible to copy-and-paste such a fractional number.

**What is the current *bug* behavior?**

- Impossible to type fractional number in the "number" field

**What is the expected *correct* behavior?**

Either
- Should be able to type fractional number in "number" field
_OR_
- Introduce "float" type or similar to hold float numbers in parallel to "number" type which will only hold integers

**Steps to reproduce**

1. Create new Workflow
2. Click to add new Task
3. Choose plugin that has "number" input parameter
4. Try to type fractional number (i.e. 0.1) in the corresponding UI field

**Relevant screenshots and/or logs**

(Insert screenshots or paste any relevant logs - please use code blocks (```) to format console output,
logs, and code as it's very hard to read otherwise.)

**Environment info**

- Latest WIPP version from CI
- Safari 13.0.4, MacOS Mojave 10.14.6

**Comment by MyleneSimon** [comment]
We are aware of this issue, it is indeed really inconvenient. 
Unfortunately it seems to be coming from the library we are using to render the forms, but we will check if a new version fixing the issue is available, and open an issue on their repo otherwise.

**Comment by MyleneSimon** [comment]
Just checked, there is an issue on their repo, but it has been opened for two years.... I asked there if anyone had a good workaround.

**Comment by MyleneSimon** [comment]
Fixed in #187
